src/py21cmmc/likelihood.py

```diff
@@ -693,20 +931,22 @@ class LikelihoodPlanck(LikelihoodBase):
     required_cores = ((core.CoreCoevalModule, core.CoreLightConeModule),)
 
     # Mean and one sigma errors for the Planck constraints
-    # The Planck prior is modelled as a Gaussian: tau = 0.058 \pm 0.012
-    # (https://arxiv.org/abs/1605.03507)
-    tau_mean = 0.058
-    tau_sigma = 0.012
+    # The Planck prior is modelled as a Gaussian: tau = 0.058 \pm 0.012 (https://arxiv.org/abs/1605.03507) tau_mean = 0.058 tau_sigma = 0.012
+    # Cosmology from Planck 2018(https://arxiv.org/pdf/1807.06209.pdf)
+    # TT TE EE lowE lensing tau_mean = 0.0544 tau_sigma = 0.0073
+    # TT lowE
+    tau_mean = 0.0522
+    tau_sigma = 0.0080
```

Comment:
I can't quite follow the progression of thought in these comments, and why `tau_mean` and `tau_sigma` have been changed.

Reply:
Sorry, this was meant to be a temporary change since I was using a different set of the cosmological parameters in Planck2018. I will change it back to the original value. But do you think we can set tau_mean and tau_sigma as optional arguments?

Reply:
Yes, we can set them as arguments in `__init__`, that will be better.